src/main/features/claim/routes/pay.ts

```diff
@@ -93,10 +95,11 @@ async function successHandler (res, next) {
       logger.error(`missing consent not given role for user, User Id : ${user.id}`)
     }
 
-    let features: string
-    if (await featureTogglesClient.isFeatureToggleEnabled(user, roles, 'cmc_admissions')) {
+    let features: string = ''
+    const featureKeys: string = config.get<string>('launchDarkly.featureFlags.admissions')
+    launchDarklyClient.callFeatureFlag(user, featureKeys , () => {
```

Comment:
is this synchronous or async?
i.e. will it return before the next code block is executed?

Reply:
make function to be Async now.